src/components/HomePage.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <Header />
+  <Carousel />
+  <BookingAvailability />
+
+  <h1>OUR ROOMS</h1>
+  <div class="room-section">
+    <div v-for="room in rooms" :key="room.id">
+      <RoomCard :data="room" />
```

Comment:
use one room card and make it horizontal or vertical using a prop horizontal: true/false